apps/tx-signer/env/.env.functional.test

```
@@ -0,0 +1,6 @@
+NETWORK=sandbox
+REST_API_NODE_URL=https://api.sandbox-2.aksh.pw:443
+RPC_NODE_ENDPOINT=https://rpc.sandbox-2.aksh.pw:443
+LOG_LEVEL=debug
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOG_LEVEL key should go before the NETWORK key

(UnorderedKey)